components/Chips/src/MDCChipView.h

```diff
@@ -151,7 +151,18 @@
 
  Default value is YES.
  */
-@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;
+@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:)
+    BOOL mdc_legacyFontScaling __deprecated;
+
+/**
+ Enable font scaling curves fallback behavior for Dynamic Type.
+
+ Scale font by calling the older [UIFont mdc_fontSizedForMaterialTextStyle:scaledForDynamicType:]
+ category method when the font is not associated with a scaling curve.
+
+ Default value is YES.
+ */
+@property(nonatomic, assign) BOOL fontScalingShouldFallback;
```

Comment:
I think it will be beneficial to connect this flag to `mdc_adjustsFontForContentSizeCategory` in some manner, so perhaps `adjustsFontForContentSizeCategoryEvenWithoutScaledFonts` or some variant therein with a suffix that clearly describes the additional behavior it enables.

Reply:
Renamed to `adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable`. Addressed in 76b80a198ed110c7e58dd3a4cfaf1d7e3f949cf7.